pkg/storage/sqlstorage/pagination.go

```diff
@@ -25,3 +25,10 @@ type AccPaginationToken struct {
 	BalanceFilter         string                  `json:"balance,omitempty"`
 	BalanceOperatorFilter storage.BalanceOperator `json:"balance_operator,omitempty"`
 }
+
+type BalancesPaginationToken struct {
```

Comment:
No need for a new token, the `AccPaginationToken` is well suited :)

Reply:
Not the same pagination token (even if properties are the same), i think the BalancesPaginationToken makes totally sense.